runtime/src/commitment.rs

```diff
@@ -41,7 +40,8 @@ pub struct BlockCommitmentCache {
     block_commitment: HashMap<Slot, BlockCommitment>,
     highest_confirmed_root: Slot,
     total_stake: u64,
-    bank: Arc<Bank>,
+    /// The slot of the bank from which all other slots were calculated.
```

Comment:
@CriesofCarrots, how's that description?

Reply:
Looks good